tools/topology/topology1/sof-mt8195-mt6359-rt1019-rt5682.m4

```diff
@@ -51,17 +51,17 @@ PIPELINE_PCM_ADD(
 	48000, 48000, 48000)
 
 # Low Latency capture pipeline 3 on PCM 18 using max 2 channels of s16le
-# Set 10000us deadline with priority 0 on core 0
+# Set 1000us deadline with priority 0 on core 0
 PIPELINE_PCM_ADD(sof/pipe-passthrough-capture.m4,
 	3, 18, 2, s16le,
-	10000, 0, 0,
+	1000, 0, 0,
 	48000, 48000, 48000)
 
 # Low Latency playback pipeline 4 on PCM 19 using max 2 channels of s16le
```

Comment:
```suggestion
# Low Latency capture pipeline 4 on PCM 19 using max 2 channels of s16le
```
the comment here and L37 seems wrong to me. Might want to fix it in another commit?

Reply:
@afq984  I think I can revise them in this patch and add commit message for it. Thanks